Apps/Src/Tasks.c

```diff
@@ -1,171 +1,202 @@
 /**
  * @copyright Copyright (c) 2018-2023 UT Longhorn Racing Solar
  * @file Tasks.c
- * @brief 
- * 
+ * @brief
+ *
  */
 
 #include "Tasks.h"
-#include "os.h"
-#include "CANbus.h"
+
+#include "CanBus.h"
 #include "Contactors.h"
 #include "Display.h"
 #include "Minions.h"
 #include "Pedals.h"
+#include "ReadCarCan.h"
 #include "ReadTritium.h"
-#include "ReadCarCAN.h"
 #include "UpdateDisplay.h"
-
+#include "os.h"
 
 /**
  * TCBs
  */
-OS_TCB Init_TCB;
-OS_TCB SendTritium_TCB;
-OS_TCB ReadCarCAN_TCB;
-OS_TCB UpdateDisplay_TCB;
-OS_TCB ReadTritium_TCB;
-OS_TCB SendCarCAN_TCB;
-OS_TCB DebugDump_TCB;
-OS_TCB CommandLine_TCB;
+OS_TCB init_tcb;
+OS_TCB send_tritium_tcb;
+OS_TCB read_car_can_tcb;
+OS_TCB update_display_tcb;
+OS_TCB read_tritium_tcb;
+OS_TCB send_car_can_tcb;
+OS_TCB debug_dump_tcb;
+OS_TCB command_line_tcb;
 
-task_trace_t PrevTasks;
+TaskTrace prev_tasks;
 
 /**
  * Stacks
  */
-CPU_STK Init_Stk[TASK_INIT_STACK_SIZE];
-CPU_STK SendTritium_Stk[TASK_SEND_TRITIUM_STACK_SIZE];
-CPU_STK ReadCarCAN_Stk[TASK_READ_CAR_CAN_STACK_SIZE];
-CPU_STK UpdateDisplay_Stk[TASK_UPDATE_DISPLAY_STACK_SIZE];
-CPU_STK ReadTritium_Stk[TASK_READ_TRITIUM_STACK_SIZE];
-CPU_STK SendCarCAN_Stk[TASK_SEND_CAR_CAN_STACK_SIZE];
-CPU_STK DebugDump_Stk[TASK_DEBUG_DUMP_STACK_SIZE];
-CPU_STK CommandLine_Stk[TASK_COMMAND_LINE_STACK_SIZE];
-
-// Variables to store error codes, stored and cleared in task error assert functions
-error_code_t Error_ReadCarCAN = /*READCARCAN_ERR_NONE*/ 0; // TODO: change this back to the error 
-error_code_t Error_ReadTritium = T_NONE;  // Initialized to no error
-error_code_t Error_UpdateDisplay = UPDATEDISPLAY_ERR_NONE;
-
-extern const pinInfo_t PININFO_LUT[]; // For GPIO writes. Externed from Minions Driver C file.
+CPU_STK init_stk[TASK_INIT_STACK_SIZE];
+CPU_STK send_tritium_stk[TASK_SEND_TRITIUM_STACK_SIZE];
+CPU_STK read_car_can_stk[TASK_READ_CAR_CAN_STACK_SIZE];
+CPU_STK update_display_stk[TASK_UPDATE_DISPLAY_STACK_SIZE];
+CPU_STK read_tritium_stk[TASK_READ_TRITIUM_STACK_SIZE];
+CPU_STK send_car_can_stk[TASK_SEND_CAR_CAN_STACK_SIZE];
+CPU_STK debug_dump_stk[TASK_DEBUG_DUMP_STACK_SIZE];
+CPU_STK command_line_stk[TASK_COMMAND_LINE_STACK_SIZE];
+
+// Variables to store error codes, stored and cleared in task error assert
+// functions
+ErrorCode error_read_car_can =
+    /*READCARCAN_ERR_NONE*/ 0;         // TODO: change this back to the error
```

Comment:
```suggestion
ErrorCode error_read_car_can = 0;
```
Is here a good place to make this change?

Reply:
yes